src/modules/test/main.c

```diff
@@ -6,7 +6,7 @@
 #include <math.h>
 
 #define DEFAULT_BIN_PATH "/usr/lib/osconfig"
-#define OSCONFIG_CONFIG_FILE "/etc/osconfig/osconfig.json"
+#define DEFAULT_OSCONFIG_CONFIG_FILE "/etc/osconfig/osconfig.json"
```

Comment:
Why changing this?

Reply:
Prior to that change OSCONFIG_CONFIG_FILE ("/etc/osconfig/osconfig.json") was the configuration file used by moduletest.
After that change "/etc/osconfig/osconfig.json" is the default osconfig config file hence name change.

In another words I am chaning the define name to match the usage of the defile

Reply:
That file is the OSConfig configuration file as described in the documentation.  It shall not be duplicated or forked or else. The test is only using it here to load the PnP model version. That shall not be changed.  This PR should be recalled.

Reply:
If there is a desire to simplify this and make moduletest don't read from /etc, simplest way is to remove this dependency on osconfig.json that is not truly necessary for currently active modules (it is necessary for the Firewall module which is currently paused; that module, and there can be others, validate that the client name for MMI calls follows the required template used by OSConfig). This can be done by changing GetClientName in src\modules\test\main.c when a new command line arg is provided to return a hard-coded string that does not contain the OSConfig model and version but instead just 'OSConfigModulesTest' for example. 

Please note however that there are no scenarios asking for such change and than when Firewall or other modules will be readded to test, that will be at risk. 

So best is at this time to just recall this PR, unless there is a need for fuzzing or something else that has impact on next release, in which case we need to discuss this team triage.

Thanks for your cooperation!